platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulers.java

```diff
@@ -50,7 +51,9 @@ public record PlatformSchedulers(
         @NonNull TaskScheduler<List<ConsensusRound>> linkedEventIntakeScheduler,
         @NonNull TaskScheduler<GossipEvent> eventCreationManagerScheduler,
         @NonNull TaskScheduler<StateSavingResult> signedStateFileManagerScheduler,
-        @NonNull TaskScheduler<StateSignatureTransaction> stateSignerScheduler) {
+        @NonNull TaskScheduler<StateSignatureTransaction> stateSignerScheduler,
+        @NonNull TaskScheduler<Void> applicationTransactionPrehandlerScheduler,
+        @NonNull TaskScheduler<Void> systemTransactionPrehandlerScheduler) {
```

Comment:
missing doc for `systemTransactionPrehandlerScheduler`

Reply:
added